.github/scripts/run-tests

```diff
@@ -84,8 +84,11 @@ do
         CUDA_VISIBLE_DEVICES=0,1 pytest ${CC_PYTEST_FLAGS} --junitxml=${RESULT_XML} ${TEST} || LOCAL_SUCCESS=$?
     elif [[ "${TEST}" == *"distributed/test_same_node"* ]]; then
         VLLM_TEST_SAME_HOST=1 torchrun --nproc-per-node=4 ${TEST} || LOCAL_SUCCESS=$?
+    elif [[ "${TEST}" == *"distributed/models_core/test_llm_logprobs"* ]]; then
```

Comment:
- Note: there are some complexities with the distributed tests. We cannot re-initialize vllm with TP>1 in the same process. So we cannot use `pytest.mark.parameterize` to run multiple models

- Ultimately, we should have a separate script from running distributed tests